### ISSUE TITLE
Tests failures using 2018.1 CE

### ISSUE THREAD

**Issue by etanshaul** [GoogleCloudPlatform/google-cloud-intellij#2154]
Note that this is CE only (no issues with UE).

```
com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponentTest > testInitComponent_feedbackIsDisabled FAILED
    java.lang.AssertionError: Cannot load resource: /idea/ApplicationInfo.xml
        at com.intellij.openapi.diagnostic.DefaultLogger.error(DefaultLogger.java:77)
        at com.intellij.openapi.diagnostic.Logger.error(Logger.java:153)
        at com.intellij.ide.plugins.PluginManagerCore.initPlugins(PluginManagerCore.java:1492)
        at com.intellij.ide.plugins.PluginManagerCore.getPlugins(PluginManagerCore.java:130)
        at com.intellij.ide.plugins.PluginManagerCore.getPlugins(PluginManagerCore.java:124)
        at com.intellij.ide.plugins.PluginManager.getPlugin(PluginManager.java:221)
        at com.google.cloud.tools.intellij.startup.GoogleAccountPluginUninstaller.uninstallIfPresent(GoogleAccountPluginUninstaller.java:47)
        at com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponent.initComponent(CloudToolsPluginInitializationComponent.java:65)
        at com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponentTest.testInitComponent_feedbackIsDisabled(CloudToolsPluginInitializationComponentTest.java:77)

        Caused by:
        java.lang.RuntimeException: Cannot load resource: /idea/ApplicationInfo.xml
            at com.intellij.openapi.application.impl.ApplicationInfoImpl.<init>(ApplicationInfoImpl.java:216)
            at com.intellij.openapi.application.impl.ApplicationInfoImpl.getShadowInstance(ApplicationInfoImpl.java:625)
            at com.intellij.ide.plugins.PluginManagerCore.checkEssentialPluginsAreAvailable(PluginManagerCore.java:1299)
            at com.intellij.ide.plugins.PluginManagerCore.initializePlugins(PluginManagerCore.java:1311)
            at com.intellij.ide.plugins.PluginManagerCore.initPlugins(PluginManagerCore.java:1486)
            ... 6 more

            Caused by:
            java.io.FileNotFoundException: /idea/ApplicationInfo.xml
                at com.intellij.openapi.util.JDOMUtil.loadDocument(JDOMUtil.java:311)
                at com.intellij.openapi.application.impl.ApplicationInfoImpl.<init>(ApplicationInfoImpl.java:212)
                ... 10 more

com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponentTest > testInitComponent_loginServiceIsInitialized FAILED
    java.lang.AssertionError: Cannot load resource: /idea/ApplicationInfo.xml
        at com.intellij.openapi.diagnostic.DefaultLogger.error(DefaultLogger.java:77)
        at com.intellij.openapi.diagnostic.Logger.error(Logger.java:153)
        at com.intellij.ide.plugins.PluginManagerCore.initPlugins(PluginManagerCore.java:1492)
        at com.intellij.ide.plugins.PluginManagerCore.getPlugins(PluginManagerCore.java:130)
        at com.intellij.ide.plugins.PluginManagerCore.getPlugins(PluginManagerCore.java:124)
        at com.intellij.ide.plugins.PluginManager.getPlugin(PluginManager.java:221)
        at com.google.cloud.tools.intellij.startup.GoogleAccountPluginUninstaller.uninstallIfPresent(GoogleAccountPluginUninstaller.java:47)
        at com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponent.initComponent(CloudToolsPluginInitializationComponent.java:65)
        at com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponentTest.testInitComponent_loginServiceIsInitialized(CloudToolsPluginInitializationComponentTest.java:104)

        Caused by:
        java.lang.RuntimeException: Cannot load resource: /idea/ApplicationInfo.xml
            at com.intellij.openapi.application.impl.ApplicationInfoImpl.<init>(ApplicationInfoImpl.java:216)
            at com.intellij.openapi.application.impl.ApplicationInfoImpl.getShadowInstance(ApplicationInfoImpl.java:625)
            at com.intellij.ide.plugins.PluginManagerCore.checkEssentialPluginsAreAvailable(PluginManagerCore.java:1299)
            at com.intellij.ide.plugins.PluginManagerCore.initializePlugins(PluginManagerCore.java:1311)
            at com.intellij.ide.plugins.PluginManagerCore.initPlugins(PluginManagerCore.java:1486)
            ... 6 more

            Caused by:
            java.io.FileNotFoundException: /idea/ApplicationInfo.xml
                at com.intellij.openapi.util.JDOMUtil.loadDocument(JDOMUtil.java:311)
                at com.intellij.openapi.application.impl.ApplicationInfoImpl.<init>(ApplicationInfoImpl.java:212)
                ... 10 more

com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponentTest > testInitComponent_usageTrackingIsEnabled FAILED
    java.lang.AssertionError: Cannot load resource: /idea/ApplicationInfo.xml
        at com.intellij.openapi.diagnostic.DefaultLogger.error(DefaultLogger.java:77)
        at com.intellij.openapi.diagnostic.Logger.error(Logger.java:153)
        at com.intellij.ide.plugins.PluginManagerCore.initPlugins(PluginManagerCore.java:1492)
        at com.intellij.ide.plugins.PluginManagerCore.getPlugins(PluginManagerCore.java:130)
        at com.intellij.ide.plugins.PluginManagerCore.getPlugins(PluginManagerCore.java:124)
        at com.intellij.ide.plugins.PluginManager.getPlugin(PluginManager.java:221)
        at com.google.cloud.tools.intellij.startup.GoogleAccountPluginUninstaller.uninstallIfPresent(GoogleAccountPluginUninstaller.java:47)
        at com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponent.initComponent(CloudToolsPluginInitializationComponent.java:65)
        at com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponentTest.testInitComponent_usageTrackingIsEnabled(CloudToolsPluginInitializationComponentTest.java:88)

        Caused by:
        java.lang.RuntimeException: Cannot load resource: /idea/ApplicationInfo.xml
            at com.intellij.openapi.application.impl.ApplicationInfoImpl.<init>(ApplicationInfoImpl.java:216)
            at com.intellij.openapi.application.impl.ApplicationInfoImpl.getShadowInstance(ApplicationInfoImpl.java:625)
            at com.intellij.ide.plugins.PluginManagerCore.checkEssentialPluginsAreAvailable(PluginManagerCore.java:1299)
            at com.intellij.ide.plugins.PluginManagerCore.initializePlugins(PluginManagerCore.java:1311)
            at com.intellij.ide.plugins.PluginManagerCore.initPlugins(PluginManagerCore.java:1486)
            at com.intellij.ide.plugins.PluginManagerCore.getPlugins(PluginManagerCore.java:130)
            at com.intellij.ide.plugins.PluginManagerCore.getPlugins(PluginManagerCore.java:124)
            at com.intellij.ide.plugins.PluginManager.getPlugin(PluginManager.java:221)
            at com.google.cloud.tools.intellij.startup.GoogleAccountPluginUninstaller.uninstallIfPresent(GoogleAccountPluginUninstaller.java:47)
            at com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponent.initComponent(CloudToolsPluginInitializationComponent.java:65)
            at com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponent$MockitoMock$214027090.initComponent$accessor$3HKpaixT(Unknown Source)
            at com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponent$MockitoMock$214027090$auxiliary$9jIrPCpk.call(Unknown Source)
            at org.mockito.internal.invocation.RealMethod$FromCallable$1.call(RealMethod.java:40)
            at org.mockito.internal.invocation.RealMethod$FromBehavior.invoke(RealMethod.java:62)
            at org.mockito.internal.invocation.InterceptedInvocation.callRealMethod(InterceptedInvocation.java:127)
            at org.mockito.internal.stubbing.answers.CallsRealMethods.answer(CallsRealMethods.java:43)
            at org.mockito.Answers.answer(Answers.java:100)
            at org.mockito.internal.handler.MockHandlerImpl.handle(MockHandlerImpl.java:104)
            at org.mockito.internal.handler.NullResultGuardian.handle(NullResultGuardian.java:29)
            at org.mockito.internal.handler.InvocationNotifierHandler.handle(InvocationNotifierHandler.java:35)
            at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor.doIntercept(MockMethodInterceptor.java:63)
            at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor.doIntercept(MockMethodInterceptor.java:49)
            at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor$DispatcherDefaultingToRealMethod.interceptSuperCallable(MockMethodInterceptor.java:110)
            at com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponent$MockitoMock$214027090.initComponent(Unknown Source)
            ... 1 more

            Caused by:
            java.io.FileNotFoundException: /idea/ApplicationInfo.xml
                at com.intellij.openapi.util.JDOMUtil.loadDocument(JDOMUtil.java:311)
                at com.intellij.openapi.application.impl.ApplicationInfoImpl.<init>(ApplicationInfoImpl.java:212)
                ... 24 more

com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponentTest > testInitComponent_feedbackIsEnabled FAILED
    java.lang.AssertionError: Cannot load resource: /idea/ApplicationInfo.xml
        at com.intellij.openapi.diagnostic.DefaultLogger.error(DefaultLogger.java:77)
        at com.intellij.openapi.diagnostic.Logger.error(Logger.java:153)
        at com.intellij.ide.plugins.PluginManagerCore.initPlugins(PluginManagerCore.java:1492)
        at com.intellij.ide.plugins.PluginManagerCore.getPlugins(PluginManagerCore.java:130)
        at com.intellij.ide.plugins.PluginManagerCore.getPlugins(PluginManagerCore.java:124)
        at com.intellij.ide.plugins.PluginManager.getPlugin(PluginManager.java:221)
        at com.google.cloud.tools.intellij.startup.GoogleAccountPluginUninstaller.uninstallIfPresent(GoogleAccountPluginUninstaller.java:47)
        at com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponent.initComponent(CloudToolsPluginInitializationComponent.java:65)
        at com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponentTest.testInitComponent_feedbackIsEnabled(CloudToolsPluginInitializationComponentTest.java:70)

        Caused by:
        java.lang.RuntimeException: Cannot load resource: /idea/ApplicationInfo.xml
            at com.intellij.openapi.application.impl.ApplicationInfoImpl.<init>(ApplicationInfoImpl.java:216)
            at com.intellij.openapi.application.impl.ApplicationInfoImpl.getShadowInstance(ApplicationInfoImpl.java:625)
            at com.intellij.ide.plugins.PluginManagerCore.checkEssentialPluginsAreAvailable(PluginManagerCore.java:1299)
            at com.intellij.ide.plugins.PluginManagerCore.initializePlugins(PluginManagerCore.java:1311)
            at com.intellij.ide.plugins.PluginManagerCore.initPlugins(PluginManagerCore.java:1486)
            ... 6 more

            Caused by:
            java.io.FileNotFoundException: /idea/ApplicationInfo.xml
                at com.intellij.openapi.util.JDOMUtil.loadDocument(JDOMUtil.java:311)
                at com.intellij.openapi.application.impl.ApplicationInfoImpl.<init>(ApplicationInfoImpl.java:212)
                ... 10 more

com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponentTest > testInitComponent_usageTrackingIsDisabled FAILED
    java.lang.AssertionError: Cannot load resource: /idea/ApplicationInfo.xml
        at com.intellij.openapi.diagnostic.DefaultLogger.error(DefaultLogger.java:77)
        at com.intellij.openapi.diagnostic.Logger.error(Logger.java:153)
        at com.intellij.ide.plugins.PluginManagerCore.initPlugins(PluginManagerCore.java:1492)
        at com.intellij.ide.plugins.PluginManagerCore.getPlugins(PluginManagerCore.java:130)
        at com.intellij.ide.plugins.PluginManagerCore.getPlugins(PluginManagerCore.java:124)
        at com.intellij.ide.plugins.PluginManager.getPlugin(PluginManager.java:221)
        at com.google.cloud.tools.intellij.startup.GoogleAccountPluginUninstaller.uninstallIfPresent(GoogleAccountPluginUninstaller.java:47)
        at com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponent.initComponent(CloudToolsPluginInitializationComponent.java:65)
        at com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponentTest.testInitComponent_usageTrackingIsDisabled(CloudToolsPluginInitializationComponentTest.java:98)

        Caused by:
        java.lang.RuntimeException: Cannot load resource: /idea/ApplicationInfo.xml
            at com.intellij.openapi.application.impl.ApplicationInfoImpl.<init>(ApplicationInfoImpl.java:216)
            at com.intellij.openapi.application.impl.ApplicationInfoImpl.getShadowInstance(ApplicationInfoImpl.java:625)
            at com.intellij.ide.plugins.PluginManagerCore.checkEssentialPluginsAreAvailable(PluginManagerCore.java:1299)
            at com.intellij.ide.plugins.PluginManagerCore.initializePlugins(PluginManagerCore.java:1311)
            at com.intellij.ide.plugins.PluginManagerCore.initPlugins(PluginManagerCore.java:1486)
            at com.intellij.ide.plugins.PluginManagerCore.getPlugins(PluginManagerCore.java:130)
            at com.intellij.ide.plugins.PluginManagerCore.getPlugins(PluginManagerCore.java:124)
            at com.intellij.ide.plugins.PluginManager.getPlugin(PluginManager.java:221)
            at com.google.cloud.tools.intellij.startup.GoogleAccountPluginUninstaller.uninstallIfPresent(GoogleAccountPluginUninstaller.java:47)
            at com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponent.initComponent(CloudToolsPluginInitializationComponent.java:65)
            at com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponent$MockitoMock$214027090.initComponent$accessor$3HKpaixT(Unknown Source)
            at com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponent$MockitoMock$214027090$auxiliary$9jIrPCpk.call(Unknown Source)
            at org.mockito.internal.invocation.RealMethod$FromCallable$1.call(RealMethod.java:40)
            at org.mockito.internal.invocation.RealMethod$FromBehavior.invoke(RealMethod.java:62)
            at org.mockito.internal.invocation.InterceptedInvocation.callRealMethod(InterceptedInvocation.java:127)
            at org.mockito.internal.stubbing.answers.CallsRealMethods.answer(CallsRealMethods.java:43)
            at org.mockito.Answers.answer(Answers.java:100)
            at org.mockito.internal.handler.MockHandlerImpl.handle(MockHandlerImpl.java:104)
            at org.mockito.internal.handler.NullResultGuardian.handle(NullResultGuardian.java:29)
            at org.mockito.internal.handler.InvocationNotifierHandler.handle(InvocationNotifierHandler.java:35)
            at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor.doIntercept(MockMethodInterceptor.java:63)
            at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor.doIntercept(MockMethodInterceptor.java:49)
            at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor$DispatcherDefaultingToRealMethod.interceptSuperCallable(MockMethodInterceptor.java:110)
            at com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponent$MockitoMock$214027090.initComponent(Unknown Source)
            ... 1 more

            Caused by:
            java.io.FileNotFoundException: /idea/ApplicationInfo.xml
                at com.intellij.openapi.util.JDOMUtil.loadDocument(JDOMUtil.java:311)
                at com.intellij.openapi.application.impl.ApplicationInfoImpl.<init>(ApplicationInfoImpl.java:212)

```

**Comment by rharter** [comment]
I've just run into this same stack trace on launch with IU 2018.2 RC.

**Comment by etanshaul** [comment]
Thanks for reporting this. I will test this out. If indeed the same thing is now happening in the EAP then its likely that the same fix (#2200) will fix this as well. 

The fix has not yet made a release, but one is planned in the next few days.

**Comment by etanshaul** [comment]
Oh, and to unblock you (assuming my previous statement holds true), you can always pull a nightly release of the plugin to get the latest changes. See the info here to pull the nightly release: https://github.com/GoogleCloudPlatform/google-cloud-intellij#pre-releases (make sure to replace `alpha` with `nightly`).

**Comment by rharter** [comment]
Thanks, will check it out.

**Comment by etanshaul** [comment]
@rharter I wasn't able to reproduce this in the latest EAP (IU-182.3684.40) with the latest release version of the Cloud Tools plugin (18.6.1). I wonder if a new version of IDEA was put out that fixed this. 

In any case, let us know if this is still a problem and nightly release hasn't solved it.

**Comment by etanshaul** [comment]
@rharter just an update that plugin version 18.7.1 was just published. This version contains the fix mentioned above (#2200). So you should be able to now test it out on the regular non-alpha channel.
